SportsHub/SportsHub.Application/Authentication/Models/DTOs/CreateArticleDTO.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace SportsHub.AppService.Authentication.Models.DTOs
+{
+    public class CreateArticleDTO
+    {
+        public int ArticleId { get; set; }
```

Comment:
ArticleId should be auto-generated, delete that field

Reply:
Deleted.